packages/ckeditor5-link/tests/linkui.js

```diff
@@ -1950,37 +2044,229 @@ describe( 'LinkUI', () => {
 				it( 'should gather information about manual decorators', () => {
 					const executeSpy = testUtils.sinon.spy( editor, 'execute' );
 
-					setModelData( model, 'f[<$text linkHref="url" linkIsFoo="true">ooba</$text>]r' );
+					setModelData( model, 'f[<$text linkHref="url" foo="bar">ooba</$text>]r' );
```

Comment:
I'm not sure if this should not be adjusted so we could notice that some manual decorator is enabled.